WMF Framework/Configuration.swift

```diff
@@ -108,6 +108,17 @@ public class Configuration: NSObject {
         return builder.components(queryParameters: queryParameters)
     }
 
+    @objc(mobileMediaWikiAPIURLComponentsForHost:withQueryParameters:)
```

Comment:
This can be removed - it was for Wikipedia Zero. The desktop URL can always be used.